learn/work/calibration-splits/index.qmd

```diff
@@ -0,0 +1,478 @@
+
+Since we want to have a calibration split similar in spirit, we are splitting the (outer) analysis set into (inner) analysis set and calibration set with the same proportion of observations going into the analysis set: 1 - 1/v.
+
+This tidymodels-internal split of the (outer) analysis set that happens automatically during resampling is implemented in `internal_calibration_split()`. This function has methods for various types of resamples. They are exported to be used by tune but not intended to be user-facing. We are showing them here for illustration purposes.
+
+```{r}
+#| label: vfold
+
+set.seed(11)
+r_set <- vfold_cv(warpbreaks, v = 5)
+r_split <- get_rsplit(r_set, 1)
+r_split
+
+# proportion of observations allocated for fitting the model in the outer split
+nrow(analysis(r_split)) / nrow(warpbreaks)
+
+split_cal <- internal_calibration_split(r_split, .get_split_args(r_set))
+split_cal
+
+# similar proportion of observations allocated for fitting the model in the inner split
+nrow(analysis(split_cal)) / nrow(analysis(r_split))
+```
+
+For most types of resamples, these principles are straightforward to apply. However, for resamples of ordered observations (like time series data) and bootstrap samples, we need to consider additional aspects.
+
+## Splits for ordered observations
+
+rsample's functions for creating resamples of ordered observations are the `sliding_*()` family: `sliding_window()`, `sliding_index()`, and `sliding_period()`.
+
+- `sliding_window()` operates on the ordered rows, allowing us to specify how far to look back to create an analysis set, how far to look ahead to create an assessment set and then sliding that across the data to create a set of resamples.
+- `sliding_index()` works in much the same way, with the key difference being that what we are sliding over aren't the rows directly but rather an index. This is useful when dealing with irregular series of data: while the number of rows which fall into a window can vary, the window length is the same across all resamples.
+- `sliding_period()` takes the idea of the `sliding_index()` function one step further and aggregates the index into periods before sliding across. This is useful for aggregating, e.g., daily data into months and then defining the analysis and assessment set in terms of months rather than the daily index directly.
+
+### Row-based splitting
+
+Let's start with the row-based splitting done by `sliding_window()`. We'll use a very small example dataset. This will make it easier to illustrate how the different subsets of the data are created but note that it is too small for real-world purposes. Let's use a data frame with 11 rows and say we want to use 5 for the analysis set, 3 for the assessment set, and leave a gap of 2 in between those two sets. We can make two such resamples from our data frame.
+
+![](images/calibration-split-window.jpg)
```

Comment:
could we do 4 for assessment set and a gap of 1?

right now there is very little air inside the assessment set with regards to the text